src/createServer.js

```diff
@@ -1,3 +1,50 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const errorMessages = require('./errorMessages');
```

Comment:
you can make a destructuring from this object
```suggestion
const { 
  missingText, 
  missingTargetCase, 
  wrongTargetCase 
} = require('./errorMessages');
```